android/quest/src/test/java/org/smartregister/fhircore/quest/ui/login/LoginViewModelTest.kt

```diff
@@ -617,8 +620,20 @@ internal class LoginViewModelTest : RobolectricTest() {
 
   @Test
   fun testForgotPasswordLoadsContact() {
-    loginViewModel.forgotPassword()
-    Assert.assertEquals("tel:0123456789", loginViewModel.launchDialPad.value)
+    val context = ApplicationProvider.getApplicationContext<Context>()
+    val launchDialPadObserver =
+      Observer<String?> { dialPadUri ->
+        if (dialPadUri != null) {
+          Assert.assertEquals("tel:1234567890", dialPadUri)
```

Comment:
Where is this `1234567890` set? It was hard coded previously. Is it in the `application_config.json`? Otherwise, this test should fail.

Reply:
It is in the app_config.